### ISSUE TITLE
Incompatibility with DynamicSurroundings

### ISSUE THREAD

**Issue by pdoktor** [UnJust1ce/Mixtape#31]
This mod is incompatible with DynamicSurroundings/Mods that change the delay between songs, causing the log to spam with the following line:

[Render thread/INFO]: Play Music: minecraft:music.menu{newSystem false, range 16.000000}

Removing mixtape fixes the issue. This is on mixtapes end.

**Comment by AliahX** [comment]
Yeah this mod is incompatible with mods that change the delay between songs, because this mod changes the delay between songs, and two mods can not do it at the same time.
